userbenchmark/optim/__init__.py

```diff
@@ -22,6 +23,19 @@
 run_on_subset: bool = False
 ignore_skips: bool = False
 
+# Models that are unstable in torch-nightly should not run in the optim world either
+def get_unstable_models() -> List[str]:
+    unstable_models: List[str]= []
+    yaml_file_path = REPO_PATH.joinpath('userbenchmark/torch-nightly/v3-cuda-tests.yaml')
+    with open(yaml_file_path, "r") as yf:
+        config_obj = yaml.safe_load(yf)
+    for d in config_obj['cuda']:
+        if not d['stable']:
+            unstable_models.append(d['model'])
```

Comment:
We may want to remove the duplicate ones because the original yaml file is on the test level, not model level.
So there might be multiple entries for a single model (for example, one entry for cuda_train and another for cuda_eval).